sonic_platform_base/sonic_xcvr/fields/xcvr_field.py

```diff
@@ -86,6 +87,7 @@ def __init__(self, name, bitpos, offset=None, **kwargs):
         super(RegBitField, self).__init__(name, offset, **kwargs)
         assert bitpos < 64
         self.bitpos = bitpos
+        self.bitmask = 1 << self.bitpos
```

Comment:
Are we using bitmask in RegBitField ?

Reply:
@andywongarista not directly, but when RegBitField is grouped inside a NumberRegField or CodeRegField

Reply:
@prgeor ok. Thanks